src/constants.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   DIAL_TIMEOUT: 30e3, // How long in ms a dial attempt is allowed to take
   MAX_COLD_CALLS: 50, // How many dials w/o protocols that can be queued
   MAX_PARALLEL_DIALS: 100, // Maximum allowed concurrent dials
+  PER_PEER_LIMIT: 4, // Allowed parallel dials per DialRequest
```

Comment:
I think it would be more coherent with the remaining names

```suggestion
  MAX_PER_PEER_DIALS: 4, // Allowed parallel dials per DialRequest
```